templates/ClientInterface.php.twig

```diff
@@ -14,25 +13,20 @@ interface ClientInterface {
      * {{ method.description|paramDescription(2)|striptags|raw }}
      *
 {% for parameter in method.parameters|sortByOptional %}
-     * @param {{  parameter.type }} ${{ parameter.name|camelize }}
+     * @param {{ renderType(parameter.type, namespace, true) }} ${{ parameter.name|camelize }}
      *        {{ parameter.description|paramDescription(6)|raw }}
      *
 {% endfor %}
      * @return ResponseInterface
      */
     public function {{ method.name }}(
-{% for parameter in method.parameters|sortByOptional %}
-{% if parameter.is_collection %}
-        array ${{ parameter.name|camelize }}{% if not parameter.required %} = null{% endif %}{% if loop.index < method.parameters|length %},{% endif %}
-
-{% elseif parameter.is_object and parameter.is_multiple_types == false %}
-        {{ parameter.type }} ${{ parameter.name|camelize }}{% if not parameter.required %} = null{% endif %}{% if loop.index < method.parameters|length %},{% endif %}
+    {% for parameter in method.parameters|sortByOptional %}
 
-{% else %}
-        ${{ parameter.name|camelize }}{% if not parameter.required %} = null{% endif %}{% if loop.index < method.parameters|length %},{% endif %}
+        {{ renderType(parameter.type, namespace, false) }} ${{ parameter.name|camelize }}
```

Comment:
In the case where there is no type for some of the parameters, the result will look like
```php
public function someMethod(
    string $param1,
     $param2WithExtraneousWhitespace,
    int $param3
): ResponseInterface;
```

Please update this so that it doesn't have the additional whitespace.

I believe there are a couple other places that might have this same issue.